src/parserindexer/indexer.py

```diff
@@ -43,20 +43,30 @@ def parse_lpsc_from_path(path_str):
         return doc_id, year, doc_url
     return None, None
 
-def map_basic(doc, noalter_prefix="ner"):
+
+# KW: indexer.py isn't handling the 'sentences' field correctly;
+# Solr ingestion fails.  I'm not sure we need it.  
+# So for now, omitting it.
+def map_basic(doc, noalter_prefix=["ner","rel"], nomap_prefix=["sentences"]):
```

Comment:
Yes, `sentences` is a dictionary.  Since we are already storing the named entities and excerpts for relations, I don't currently see a need to store `sentences` in Solr.